english/Krishna/krishna.md

```diff
@@ -0,0 +1,11 @@
+Krishna is the Supreme Personality of Godhead.
+Our natural tendency is to love krishna but it is covered by maya.
+Chanting of the mantra "Hare Krishna Hare Krishna Krishna Krishna Hare Hare
+Hare Rama Hare Rama Rama Rama Hare Hare" will remove that covering.
+And we will go back home back to godhead.
+
+Hare Krishna Hare Krishna Krishna Krishna Hare Hare
+
+Hare Rama Hare Rama Rama Rama Hare Hare
+
+click [here](iskcondesiretree.com) to know more about Krishna.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.